pom.xml

```diff
@@ -91,6 +91,10 @@
                 <configuration>
                     <excludes>
                         <exclude>**/Main*</exclude>
+                        <exclude>**/Fruit*</exclude>
+                        <exclude>**/Operation*</exclude>
+                        <exclude>**/Transaction*</exclude>
+                        <exclude>**/ProcessDataServiceImpl*</exclude>
```

Comment:
```suggestion
```
let's test this service, if it is required for test coverage

Reply:
I think no need , because inside only solo for-each loop